statistics/table.go

```diff
@@ -197,6 +197,24 @@ func (t *Table) ColumnByName(colName string) *Column {
 	return nil
 }
 
+func (t *Table) getColumnStatsInfo(colID int64) (*Histogram, *CMSketch, *TopN) {
+	colStatsInfo := t.Columns[colID]
+	return colStatsInfo.Histogram.Copy(), colStatsInfo.CMSketch.Copy(), colStatsInfo.TopN.Copy()
+}
+
+func (t *Table) getIndexStatsInfo(idxID int64) (*Histogram, *CMSketch, *TopN) {
+	idxStatsInfo := t.Indices[idxID]
+	return idxStatsInfo.Histogram.Copy(), idxStatsInfo.CMSketch.Copy(), idxStatsInfo.TopN.Copy()
+}
+
+// GetStatsInfo returns their statistics according to the ID of the column or index, including histogram, CMSketch and TopN.
+func (t *Table) GetStatsInfo(ID int64, isIndex int) (*Histogram, *CMSketch, *TopN) {
+	if isIndex == 0 {
+		return t.getColumnStatsInfo(ID)
+	}
+	return t.getIndexStatsInfo(ID)
+}
+
```

Comment:
`getColumnStatsInfo` and `getIndexStatsInfo` are simple and not used by other functions directly, so how about merging these 3 functions into 1?